plugin-server/package.json

```diff
@@ -29,6 +29,9 @@
         "services:clean": "cd .. && docker compose -f docker-compose.dev.yml rm -v",
         "services": "pnpm services:stop && pnpm services:clean && pnpm services:start"
     },
+    "graphile-worker": {
+        "maxContiguousErrors": 300
+    },
```

Comment:
It look me a minute to realize cosmiconfig will load from this. Weird!

Reply:
Yeah, welcome back to `java.properties` without documentation not convention. Cannot add a comment here, that's why I added a `KLUDGE` comment on the runner init, to point here.